lib/cart.rb

```diff
@@ -0,0 +1,15 @@
+require_relative "menu.rb"
+
+class Cart
```

Comment:
The add_to_cart method in this class could be moved to the Order class - but this is a bit subjective, so I'm not sure if my advice is the best here!